pybossa/forms/forms.py

```diff
@@ -129,6 +129,36 @@ class ProjectSyncForm(Form):
     target_key = TextField(lazy_gettext('API Key'))
 
 
+class ProjectQuizForm(Form):
+    enabled = BooleanField(lazy_gettext('Enable Quiz Mode'))
+    questions_per_quiz = IntegerField(
+        lazy_gettext('Number of questions per quiz'),
+        [
+            validators.Required(lazy_gettext('This field must be a positive integer.')),
+            validators.NumberRange(min=1)
+        ]
+    )
+    correct_answers_to_pass = IntegerField(
+        lazy_gettext('Number of correct answers to pass quiz'),
+        [
+            validators.Required(lazy_gettext('This field must be a positive integer.')),
+            validators.NumberRange(min=1)
+        ]
+    )
+
+    def validate_correct_answers_to_pass(form, field):
```

Comment:
where's this validator used? fyi, custom validator cand be added and called from forms/validator.py

Reply:
It's the built-in "inline-validator" behavior of WTForms. Any method name that matches `validate_<fieldname>` is a validator.

Reply:
gotcha, thanks